thorlcr/activities/countproject/thcountprojectslave.cpp

```diff
@@ -186,9 +187,9 @@ class CountProjectActivity : public BaseCountProjectActivity, implements ILookAh
         ActivityTimer t(totalCycles, timeActivities);
         if (first) 
         {
-            first = false;
             prevRecCount = count = getPrevCount();
             signalNext();
+            first = false;
```

Comment:
Given start()/nextRow()/stop() should only be called by a single thread, this should not make any difference afaics.